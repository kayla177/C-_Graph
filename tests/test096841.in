ENTITY EA8KTFEX name_EA8K journal
ENTITY BK0Q1FF0 name_BK0Q institution
ENTITY JR9OR8HN name_JR9O conference
RELATIONSHIP EA8KTFEX presented_at BK0Q1FF0 4.1
PRINT EA8KTFEX
HIGHEST
EXIT
