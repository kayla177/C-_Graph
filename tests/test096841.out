success
success
success
success
BK0Q1FF0
EA8KTFEX BK0Q1FF0 4.1
